### DISCUSSION TITLE
iphone faceid - There are no matching passkeys saved in your icloud keychain for localhost

### DISCUSSION THREAD

**Discussion by bugthug404** [MasterKale/SimpleWebAuthn#520] [General]
HI community,

I've implemented webauthn on my localhost according to the example on this repo. it is working fine with macbook touchid & windows hello but not working with iphone faceid showing "there are no matching passkeys saved in your icloud keychain for localhost"

is there anything i am not aware of or any special requirement for iphone faceid?

server response for authentication-option 
<img width="658" alt="Screenshot 2024-02-13 at 12 18 40 AM" src="https://github.com/MasterKale/SimpleWebAuthn/assets/28305019/3013e131-4d7e-40c4-8216-3ddfc77bfba3">

**Comment by MasterKale** [comment]
Hello @bugthug404, those options look fine to me. Are you sure you're using the same iCloud account on both your MacBook and your iPhone? And is **Settings > iCloud (the top option that says your name) > iCloud > Passwords and Keychain > Sync this Mac/iPhone** turned on on both devices? Those are common reasons why passkeys don't get synced between devices.

Let me know what you find out and we can continue from there.
